### PR TITLE
Function96 V2: Fixed info.json file for LAYOUT_default and LAYOUT_ansi_splitspace

### PR DESCRIPTION
**Pull request by MatthieuLu**
## Description

Updated the info.json and .h files to reflect the odd switch matrix regarding the long keys on the right.

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — APPROVED** [review]

File: keyboards/function96/v2/v2.h [hunk 1]
```diff
@@ -30,10 +30,10 @@
...
 	K000, K001, K002, K003, K004,       K006, K007, K008, K009, K010, K011, K012, K013, K014, K015, K016, K017, K018, \
 	K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112,       K114, K115, K116, K117, K118, \
-	K200,       K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213, K214, K215, K216, K217,      \
-	K300,       K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312, K313, K314, K315, K316, K317, K318,\
-	K400, K401, K402, K403, K404, K405, K406, K407, K408, K409, K410, K411, K412,       K414, K415, K416, K417,      \
-	K500, K501, K502,       K504,       K506,       K508,       K510, K511, K512,       K514, K515, K516, K517, K518 \
+	K200,       K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213, K214, K215, K216, K217, K318,     \
+	K300,       K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312, K313, K314, K315, K316, K317, \
+	K400, K401, K402, K403, K404, K405, K406, K407, K408, K409, K410, K411, K412,       K414, K415, K416, K417, K518,      \
+	K500, K501, K502,       K504,       K506,       K508,       K510, K511, K512,       K514, K515, K516, K517  \
```
Comment (noroadsleft) [inline keyboards/function96/v2/v2.h:36]:
```suggestion
	K200,       K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213, K214, K215, K216, K217,      \
	K300,       K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312, K313, K314, K315, K316, K317, K318,\
	K400, K401, K402, K403, K404, K405, K406, K407, K408, K409, K410, K411, K412,       K414, K415, K416, K417,      \
	K500, K501, K502,       K504,       K506,       K508,       K510, K511, K512,       K514, K515, K516, K517, K518 \
```

File: keyboards/function96/v2/v2.h [hunk 2]
```diff
@@ -45,10 +45,10 @@
...
 	K000, K001, K002, K003, K004,       K006, K007, K008, K009, K010, K011, K012, K013, K014, K015, K016, K017, K018, \
 	K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112,       K114, K115, K116, K117, K118, \
-	K200,       K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213, K214, K215, K216, K217,      \
-	K300,       K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312,       K314, K315, K316, K317, K318,\
-	K400,       K402, K403, K404, K405, K406, K407, K408, K409, K410, K411, K412,       K414, K415, K416, K417,      \
-	K500, K501, K502,                   K506,                   K510, K511, K512,       K514, K515, K516, K517, K518\
+	K200,       K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213, K214, K215, K216, K217, K318, \
+	K300,       K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312,       K314, K315, K316, K317, \
+	K400,       K402, K403, K404, K405, K406, K407, K408, K409, K410, K411, K412,       K414, K415, K416, K417, K518, \
+	K500, K501, K502,                   K506,                   K510, K511, K512,       K514, K515, K516, K517\
```
Comment (noroadsleft) [inline keyboards/function96/v2/v2.h:51]:
```suggestion
	K200,       K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213, K214, K215, K216, K217,      \
	K300,       K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312,       K314, K315, K316, K317, K318,\
	K400,       K402, K403, K404, K405, K406, K407, K408, K409, K410, K411, K412,       K414, K415, K416, K417,      \
	K500, K501, K502,                   K506,                   K510, K511, K512,       K514, K515, K516, K517, K518\
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
The changeset you have currently actually breaks compiling from source. The changes to `v2.h` should be reverted, but the changes to `info.json` applied. This will fix Configurator compiles without breaking source code compilation.

Applying the following changes should get this where it needs to be.

**Comment by MatthieuLu** [comment]
Changes committed

**Review by fauxpark — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**